cmd/server/handler.go

```diff
@@ -201,7 +201,7 @@ func setup(ctx context.Context, d driver.Registry, cmd *cobra.Command) (admin *h
 		NewMiddlewareFromLogger(d.Logger(),
 			fmt.Sprintf("hydra/admin: %s", d.Config().IssuerURL(ctx).String()))
 	if d.Config().DisableHealthAccessLog(config.AdminInterface) {
-		adminLogger = adminLogger.ExcludePaths("/admin"+healthx.AliveCheckPath, "/admin"+healthx.ReadyCheckPath)
+		adminLogger = adminLogger.ExcludePaths(healthx.AliveCheckPath, healthx.ReadyCheckPath)
```

Comment:
Are you sure about this? The path is - if I recall correctly - `/admin/health/status`

Reply:
I am pretty sure the path is actually `/health/alive`

when testing the endpoints manually i get a proper response:
```
curl localhost:4445/health/alive   
{"status":"ok"}
```

while `localhost:4445/admin/health/status` returns a 404.

this path is also used by the helm chart: 
https://github.com/ory/k8s/blob/master/helm/charts/hydra/templates/deployment.yaml#L92-L95